src/main/java/reposense/parser/CsvParser.java

```diff
@@ -59,6 +59,11 @@ public CsvParser(Path csvFilePath) throws IOException {
 
             for (int lineNumber = 2; (line = br.readLine()) != null; lineNumber++) {
                 String[] elements = line.split(ELEMENT_SEPARATOR_REGEX);
+                for (int colNum = 0; colNum < elements.length; colNum++) {
```

Comment:
Can use `for(String element: elements)`

Reply:
Here I want to edit the contents in `elements[]` under certain conditions. So may not be appropriate to use range based loop as `element` is just an iterative copy of the value in `elements[]`?

https://stackoverflow.com/questions/15844443/why-does-the-foreach-statement-not-change-the-element-value

Reply:
Right，forgot this.